README.md

```diff
@@ -247,6 +247,9 @@ This project uses [Azure Developer CLI (`azd`)](https://aka.ms/azd) to provision
 # Login to azd. Only required once per install.
 azd auth login
 
+# Use Flag if login does not work
```

Comment:
Hi @ShatilKhan . I think maybe we can be a bit more specific about what doesn't work. I think the issue is not the login/authentication not working, but the fact that it doesn't open a browser window? A link to the official documentation where this is explained could also be a good addition.

Reply:
No problem @anfibiacreativa,  thanks for noticing the screenshot.
Glad to contribute.
Should I add in the link to the docs still? 
Let me know.

Reply:
Yes, please! I think explaining exactly what error with auth the flag is connected to, is helpful for devs that may be facing other issues, not to get confused. Thank you!

Reply:
Hey @anfibiacreativa , sorry for the delay.
As per your message, I added a link to the official doc & also a line explaining why we need the specific `flag`.
Please have a look & let me know.

Reply:
Oh I see now that you added them to this PR instead of the previous one? Is that correct?

Reply:
Yes, sorry for the confusion. My Branches got mixed up accidently.
This commit is part of this PR only.
We don't need it in the previous one.